### PR TITLE
vcredist140, vcredist2017: update to Visual Studio 2017 RTM

### PR DESCRIPTION
**Pull request by jberezanski**

### CODE REVIEW

File: manual/vcredist140/vcredist140.nuspec [hunk 1]
```diff
@@ -3,23 +3,23 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>vcredist140</id>
-    <version>14.10.24930.0-rc6</version>
+    <version>14.10.25008.0</version>
     <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/vcredist140</packageSourceUrl>
     <owners>chocolatey,jberezanski</owners>
-    <title>Microsoft Visual C++ Redistributable for Visual Studio 2017 RC (2017-02-07)</title>
+    <title>Microsoft Visual C++ Redistributable for Visual Studio 2017 RTM</title>
```
Comment (AdmiringWorm) [inline manual/vcredist140/vcredist140.nuspec]: Is it really necessary to have the RTM in the title?
Just curious.

File: manual/vcredist2017/vcredist2017.nuspec [hunk 2]
```diff
@@ -3,21 +3,21 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>vcredist2017</id>
-    <version>14.10.24930.20170211-rc6</version>
+    <version>14.10.25008.0</version>
     <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/vcredist2017</packageSourceUrl>
     <owners>chocolatey,jberezanski</owners>
-    <title>Microsoft Visual C++ Redistributable for Visual Studio 2017 RC (2017-02-07)</title>
+    <title>Microsoft Visual C++ Redistributable for Visual Studio 2017 RTM</title>
```
Comment (AdmiringWorm) [inline manual/vcredist2017/vcredist2017.nuspec]: same question as the previous comment.

**Comment by jberezanski** [comment]
"RTM" is to clearly distinguish this release from subsequent Updates (since Microsoft will probably release a few of them during the lifecycle of VS 2017, as it did in the past).

I'm not sure about renaming the file, however. The package is constructed in such a way that data.ps1 is the only file that changes between versions and it contains just data (two hashtables), there are no functions or other executable code there. I could probably have made it a .psd1.

**Comment by jberezanski** [comment]
In fact, the guideline specifically discusses helper functions:

> If you created custom helper functions put them all in the helpers.ps1 to keep installer clean and understandable

There are none in data.ps1, so in my opinion naming the file helpers.ps1 would be more confusing here.

**Comment by jberezanski** [comment]
Btw. I updated the icon.

**Comment by AdmiringWorm** [comment]
> "RTM" is to clearly distinguish this release from subsequent Updates (since Microsoft will probably release a few of them during the lifecycle of VS 2017, as it did in the past).

true, they probably will. But don't they also increment the version?

I'm not sure about renaming the file, however. The package is constructed in such a way that data.ps1 is the only file that changes between versions and it contains just data (two hashtables), there are no functions or other executable code there. I could probably have made it a .psd1.

Good point. I must have seen something else somewhere as my mind immediately thought there was a function there.


FYI, just for future notes. You don't need a new version yourself to chocolatey, we can do that when merging a PR by adding `[PUSH packageName1 packageName2]` to the Merge Title (this means it will be auto-approved when automated checks have completed)

Anyways, I'll merge this one now

**Comment by AdmiringWorm** [comment]
> Btw. I updated the icon.

I noticed.

**Comment by jberezanski** [comment]
> I'll merge this one now

Thanks!

>> "RTM" is to clearly distinguish this release from subsequent Updates (since Microsoft will probably release a few of them during the lifecycle of VS 2017, as it did in the past).

> true, they probably will. But don't they also increment the version?

Yes, they do, but I wanted the package _title_ to indicate this, without requiring the user to look at the version numer (and remember which five-digit number corresponds to the RTM release). Take a look at [vcredist140 version history](https://chocolatey.org/packages/vcredist140#versionhistory) to see what I mean.

> FYI, just for future notes. You don't need a new version yourself to chocolatey, we can do that when merging a PR (...)

Thanks, I'm aware of that. Here I needed this package on chocolatey.org as quickly as possible, because I was testing other packages dependent upon it.

**Pull request merged**